x-pack/plugins/triggers_actions_ui/public/index.ts

```diff
@@ -18,6 +18,7 @@ export type {
   RuleType,
   RuleTypeIndex,
   RuleTypeModel,
+  RuleStatus,
```

Comment:
@JiaweiWu Could you review this PR? It is a very small change that we already talked about here 

https://github.com/elastic/kibana/pull/130705#discussion_r861651924